lib/src/components/navigation bar/navigation_bar.dart

```diff
@@ -251,9 +264,11 @@ class _NavigationItem extends ZetaStatelessWidget {
       child: InkResponse(
         borderRadius: context.rounded ? Zeta.of(context).radius.rounded : Zeta.of(context).radius.none,
         onTap: onTap,
+        hoverColor: colors.surfaceHover,
+        highlightShape: BoxShape.rectangle,
         child: Semantics(
           button: true,
-          explicitChildNodes: true,
+          excludeSemantics: true,
```

Comment:
Can you revert this change please. We need ```explicitChildNodes: true,``` so that if the icon has a notification badge the semantics can see it

Reply:
When the explicitChildNodes is set to true the notification dot the notification indicator fails accessibility. In fact the default color medium notification indicator always fails accessibility due to text contrast ratios. The large however does not fail: 
![image](https://github.com/user-attachments/assets/2f9028dc-4988-4d0c-adf9-b5a7f9a9e5fc)

Reply:
The indicator fails contrast ratios only with small sizes: 
![image](https://github.com/user-attachments/assets/a42b3713-c203-491a-b3ac-381619b6b802)
I'll raise the issue to the designers

Reply:
I think there must be some other issue with the test here - the  colors being tested are  #E0E3E9 (light grey) and #FFFFFF (white) - for the notification indicator the background should be red?

Reply:
Perhaps some overlap with the border?